substrate/frame/authority-discovery/Cargo.toml

```
@@ -19,37 +19,23 @@ targets = ["x86_64-unknown-linux-gnu"]
 codec = { features = [
 	"derive",
 ], workspace = true }
-frame-support = { workspace = true }
-frame-system = { workspace = true }
+frame = { workspace = true, features = ["experimental", "runtime"] }
```

Please remove the `experimental` feature flag from now on as in #7177 
```suggestion
frame = { workspace = true, features = ["runtime"] }
```